MapboxGeocoder/MBGeocoder.swift

```diff
@@ -346,6 +356,114 @@ open class Geocoder: NSObject {
         components.queryItems = params
         return components.url!
     }
+    
+    /**
+     Send a keydown event
+    */
+    func initKeydown(options: GeocodeOptions){
```

Comment:
How meaningful is “keydown” in this context? MapboxGeocoder.swift is completely UI-agnostic, containing no UI code. A query could originate from any source, whether from the user typing or speaking or from a programmatic query.